ICSharpCode.CodeConverter/CSharp/NodesVisitor.cs

```diff
@@ -495,13 +501,37 @@ public override CSharpSyntaxNode VisitPropertyStatement(VBSyntax.PropertyStateme
                         SyntaxFactory.BracketedParameterList(SyntaxFactory.SeparatedList(node.ParameterList.Parameters.Select(p => (ParameterSyntax)p.Accept(TriviaConvertingVisitor)))),
                         accessors
                     );
-                else {
+                } else {
+                    var identifier = node.Identifier;
+                    if (accessedThroughMyClass) {
+                        var identifierName = "MyClass" + node.Identifier.ValueText;
+                        var getReturn = SyntaxFactory.Block(SyntaxFactory.ParseStatement($"return this.{identifierName};"));
+                        var getAccessor = SyntaxFactory.AccessorDeclaration(SyntaxKind.GetAccessorDeclaration, getReturn);
+                        var setValue = SyntaxFactory.Block(SyntaxFactory.ParseStatement($"this.{identifierName} = value;"));
```

Comment:
I'll add a call to ConvertIdentifier first and change the test to include some inconsistent VB casing.

Especially when working with strings, its really easy to lose track of what language they're in. You'll see I've started prefixing some variable names with `cs` or `vb` to help with this in complicated areas. I feel like the `SyntaxFactory.Parse*`  methods make it easier to get lost due to encouraging more strings. I think you've used them in moderation at a sensible level in this case, so good job!

Reply:
I skipped `ConvertIdentifier`, since it could prefix the variable with `@`, which would break this. Perhaps it should concat then convert?

I'll make sure to prefix with `cs` and `vb` in future.